provisioning/virtual-machines/README.md

```diff
@@ -1,81 +1,77 @@
 # Guide for creating and uploading VMs in CrownLabs
 
-## Upload custom VMs to the cluster
+## Create and upload custom VMs to the cluster
 
-### Dependencies
+The approach currently suggested for the creation and upload of VMs in CrownLabs involves the usage of the [setup-crownlabs-vm script](setup-crownlabs-vm/setup-crownlabs-vm.sh).
+In a nutshell, it takes care of automatically:
+- Create new VirtualBox VMs and install the guest OS (`xubuntu`);
+- Install custom software and the tools required by CrownLabs with Ansible;
+- Convert the resulting virtual HDD to the correct format and upload it to a Docker Registry.
 
-Before uploading your VM, you must run the [prepare-vm.sh](scripts/prepare-vm.sh) script from inside the VM.
-The script will install and configure:
-- **TigerVNC server**, which allows to connect to the VM desktop from a remote machine;
-- **NoVNC with websockify server**, which allows the above connection to be established through HTTP/HTTPS;
-- **Prometheus node exporter**, which exports some run-time information of the VM (e.g., CPU/memory consumption) to the Prometheus monitoring system, running on the Kubernetes cluster
-- **cloud-init**, which enables to customize some running parameters of the VM at boot time.
+Once the VM image has been correctly uploaded to the registry, it is possible to setup a new laboratory associated to it, as explained [here](../courses).
 
-To verify that the setup works, reboot the machine after running the `prepare-vm.sh` script.
-From inside the machine, start a browser and connect to page `http://localhost:6080`, using password `ccroot`.
+### TL; DR;
```

Comment:
Marco, what does it mean "TL; DR;" here?

Reply:
It should refer to the brief version of a long description. I replaced it with "Quick start guide".